### PR TITLE
LPS-167887 [CP 2.0] Fix the language key in the table

### PR DESCRIPTION
**Pull request by caiosouza15**

### CODE REVIEW

**Review by JoMaAlves — APPROVED** [review]
LGTM ​🦖​🌋​ 
Thank you @caiosouza15

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json [hunk 1]
```diff
@@ -89,6 +89,10 @@
 	"download-key": "Download Key",
 	"dubai-uae": "Dubai, UAE",
 	"dxp": "DXP",
+	"dxp-backup": "Dxp Backup",
+	"dxp-development": "Dxp Development",
+	"dxp-non-prod": "Dxp Non-Prod",
+	"dxp-prod": "Dxp Prod",
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json:95]:
```suggestion
	"dxp-backup": "DXP Backup",
	"dxp-development": "DXP Development",
	"dxp-non-prod": "DXP Non-Prod",
	"dxp-prod": "DXP Prod",
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by caiosouza15** [comment]
ci:test:sf

**Comment by caiosouza15** [comment]
ci:test:relevant

**Comment by JoMaAlves** [comment]
ci:forward

**Pull request closed without merging**